components/com_groups/admin/controllers/manage.php

```diff
@@ -357,7 +357,7 @@ public function saveTask()
 				->setLayout('edit')
 				->set('group', $group)
 				->set('customFields', $customFields)
-				->set('customAnswers', $customFieldForm)
+				->set('customAnswers', $customFieldForms)
```

Comment:
I couldn't find where 'customFieldForms' is being created in the code. The change to pluralize 'form' to 'forms' here is intentional though?

Reply:
No, good catch. I need to fix this at some point